docker-compose-3-nodes.yaml

```diff
@@ -85,13 +83,14 @@ services:
       - 8800
       - 5050
     volumes:
-      - keys:/shared_keys
```

Comment:
keygen was run for validator-2,3.. on first validator machine. Respective validators will now generate their keys instead of depending on first validator. This mount is thus not required anymore.

Reply:
I know they were there for a reason... to ensure validators start in order. How does this change impact that behavior.

Reply:
Validators doesn't need to start in order, current design limitation is to start PoET engine on genesis before validator node.
Is there a reason to start validators in order?

Reply:
yes, like I said these files are meant for tests

Reply:
@manojgop ^ comments? I changed this as part of review.

Reply:
@arsulegai We can keep this changes for now as a temporary solution. But this copying validator public key to poet engine won't work if poet engine and validator are running on different machines. I saw API get_peers_public_keys() in consensus SDK. See if we can use it to get public key of validator.